google-cloud-spanner/src/main/java/com/google/cloud/spanner/SessionPoolOptions.java

```diff
@@ -223,6 +259,135 @@ private enum ActionOnSessionLeak {
     FAIL
   }
 
+  @VisibleForTesting
+  enum ActionOnInactiveTransaction {
+    WARN,
+    WARN_AND_CLOSE,
+    CLOSE
+  }
+
+  /** Configuration options for task to clean up inactive transactions. */
+  static class InactiveTransactionRemovalOptions {
```

Comment:
"Long-Running" is one of the reasons why transaction can be inactive. Tomorrow if there are other reasons why a transaction can be in-active it will still remain abstracted in inner layers and won't be seen as a pool option.